index.Rmd

```diff
@@ -318,11 +319,11 @@ plot2=leaflet() %>%
 
 ## Methods - Raster Analysis
 
-NLCD Raster data were used to determine the percent of deciduous and mixed (deciduous/coniferous) forest that covered each county. The code to calculate this data was shown earlier, but a detailed explanation (which ignores the loop) is shown below.
+NLCD Raster data were used to determine the percent of deciduous and mixed (deciduous/coniferous) forest that covered each county. The code to calculate this data was shown earlier,  but a detailed explanation (which ignores the loop) is shown below.
```

Comment:
At this point, I have seen so much code and I'm not sure where the code to calculate the data occurred. However, then I wondered if the code following this line is what you were referring to and is just a summary of what you were talking about? I'm sorry, I'm just confused on what this code to calculate the data exactly is.

Reply:
The explanation is shown below where I break down every line of the loop.